package.json

```diff
@@ -98,9 +97,131 @@
     "vue": "^3.2.14",
     "vue-loader": "^17.0.0",
     "vue-template-compiler": "^2.5",
+    "webpack": "^5.72",
+    "webpack-cli": "^4.9.1",
     "webpack-notifier": "^1.15.0",
     "zombie": "^6.1.4"
   },
+  "peerDependencies": {
+    "@babel/plugin-proposal-class-properties": "^7.0.0",
+    "@babel/plugin-transform-react-jsx": "^7.12.11",
+    "@babel/preset-react": "^7.0.0",
+    "@babel/preset-typescript": "^7.0.0",
+    "@symfony/stimulus-bridge": "^3.0.0",
+    "@vue/babel-helper-vue-jsx-merge-props": "^1.0.0",
+    "@vue/babel-preset-jsx": "^1.0.0",
+    "@vue/compiler-sfc": "^3.0.2",
+    "eslint": "^8.0.0",
+    "eslint-webpack-plugin": "^3.1.0",
+    "file-loader": "^6.0.0",
+    "fork-ts-checker-webpack-plugin": "^7.0.0",
+    "handlebars": "^4.7.7",
+    "handlebars-loader": "^1.7.0",
+    "less": "^4.0.0",
+    "less-loader": "^11.0.0",
+    "postcss": "^8.3.0",
+    "postcss-loader": "^7.0.0",
+    "sass": "^1.17.0",
+    "sass-loader": "^13.0.0",
+    "stylus": "^0.58.1",
+    "stylus-loader": "^7.0.0",
+    "ts-loader": "^9.0.0",
+    "typescript": "^4.2.2",
+    "vue": "^3.2.14",
+    "vue-loader": "^17.0.0",
+    "vue-template-compiler": "^2.5",
+    "webpack": "^5.72",
```

Comment:
No `webpack-cli` here?

Reply:
dev dependencies of webpack have no impact when developing Encore. We are not developing webpack in such situation.

Reply:
Do I have anything to change here? ATM `webpack-cli`, in this `package.json` is present in `devDependencies` only.